crates/bevy_pbr/src/light.rs

```diff
@@ -92,6 +92,9 @@ impl Default for PointLightShadowMap {
 /// approximation for light sources VERY far away, like the sun or
 /// the moon.
 ///
+/// The light's default direction is negative-Z (0, 0, -1) and is
```

Comment:
```suggestion
/// The light shines in the forward direction of the transform (negative z) and is
```
It's true that that is the default direction, but I think it's better to explain *why* that is the case.

Reply:
@devil-ira Can you clarify this statement? Why it's negative-Z or what it has a default direction?

Reply:
I think it's important to explain "how is the direction inferred from the transform", and then tie that back to the default transform value.